mongocrypt-sys/etc/bindgen.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -x
+
+REPO_DIR=$(readlink -f $(dirname $0)/../..)
+BINDINGS_PATH=${REPO_DIR}/mongocrypt-sys/src/bindings.rs
+LIBMONGOCRYPT_DIR=${LIBMONGOCRYPT_DIR:-"${REPO_DIR}/../libmongocrypt"}
+if [ ! -f $LIBMONGOCRYPT_DIR/src/mongocrypt.h.in ]; then
+    echo 'LIBMONGOCRYPT_DIR must point to the base directory of the libmongocrypt repo'
+    exit 1
+fi
+LIBMONGOCRYPT_TAG=${LIBMONGOCRYPT_TAG:-'1.5.0-rc0'}
```

Comment:
rather than having a default, I think it might be better to just return an error LIBMONGOCRYPT_TAG isn't set. that way vendoring a version is always explicit.

Reply:
Done.